src/components/Button.tsx

```diff
@@ -204,7 +204,7 @@ const Button = (
     variant,
     disabled,
     sx: {
-      color: loading ? 'transparent !important' : 'inherit',
+      color: loading ? 'transparent !important' : null,
```

Comment:
We could also pass the value only when loading is true. Wouldn't need to resort to `null`
```suggestion
      ...(loading && { color: 'transparent !important' },
```

Reply:
yea, you're right 😄